azure/v11.4.0/README.md

```diff
@@ -0,0 +1,41 @@
+## :zap:  Giant Swarm Release 11.4.0 for Azure :zap:
```

Comment:
@giantswarm/chapter-se if your customer is on Azure, please carefully review the release notes and especially the documented post-cluster-upgrade migration process steps.

Reply:
Thanks a lot for the very detailed explanation on what to do